addOns/grpc/src/main/java/org/zaproxy/addon/grpc/HttpPanelGrpcView.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2024 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.grpc;
+
+import java.awt.BorderLayout;
+import javax.swing.JComponent;
+import javax.swing.JPanel;
+import org.apache.commons.configuration.FileConfiguration;
+import org.fife.ui.rtextarea.RTextScrollPane;
+import org.zaproxy.zap.extension.httppanel.Message;
+import org.zaproxy.zap.extension.httppanel.view.AbstractStringHttpPanelViewModel;
+import org.zaproxy.zap.extension.httppanel.view.HttpPanelView;
+import org.zaproxy.zap.extension.httppanel.view.HttpPanelViewModel;
+import org.zaproxy.zap.extension.httppanel.view.HttpPanelViewModelEvent;
+import org.zaproxy.zap.extension.httppanel.view.HttpPanelViewModelListener;
+
+public class HttpPanelGrpcView implements HttpPanelView, HttpPanelViewModelListener {
+
+    /**
+     * Default name used for {@code MessageContainer}.
+     *
+     * @see org.zaproxy.zap.view.messagecontainer.MessageContainer
+     */
+    public static final String NAME = "HttpPanelGrpcView";
+
+    private static final String CAPTION_NAME = "GRPC";
```

Comment:
gRPC

Reply:
should i rename the Name string also to :HttpPanelgRPCView, but it is not following proper conventions

Reply:
This is the name shown to the user not code.